_events/talks/event-012.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "UXF - Framework for creating Virtual Reality human behavior experiments in Unity"
+authors:
+    - &speaker name: Dr. Jack Brookes
+      email: jack.brookes@ucl.ac.uk
+      is_speaker: true
+    - name: Mr. Matthew Warburton
+    - name: Prof. Mark Mon-Williams
+    - name: Dr. Faisal Mushtaq
```

Comment:
Is it required for the layout to have affiliations at least one affiliation? 
I would propose that we could either ask the authors to provide us with correct affiliations and otherwise, we should consider leaving this as is. This is the information, that was provided to us.

Reply:
I adapted the website such that they can be optional (0ed0985)